test/test_upgrade.sh

```diff
@@ -55,7 +55,7 @@ TMP_LOCALSETTINGS="$TMP_DIR/LocalSettings.php"
 
 # Source the default env vars used for building and create a new LocalSettings file 
 set -o allexport; source ../Docker/build/Wikibase/default.env; set +o allexport
-envsubst < ../Docker/build/Wikibase/LocalSettings.php.template > "$TMP_LOCALSETTINGS"
+envsubst < "../Docker/build/Wikibase/$MEDIAWIKI_DOCKER_TEMPLATE_FILE" > "$TMP_LOCALSETTINGS"
```

Comment:
Looking at this again, this will now default to the newest template which is not what we want.

This could be resolved by either 

1. re-specify the template in the `old-versions/version.env` file 
2. we source the correct `versions/wmde.N` in the test upgrade somehow (this would only work for our releases but could be overridden with an env var in the old-version file)

Reply:
heh, looking at it another time this is what we want. this only happens when running against the built version which already specifies this value. nevermind! :facepalm: